patches/chrome-tools-build-mac-BUILD.gn.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/chrome/tools/build/mac/BUILD.gn b/chrome/tools/build/mac/BUILD.gn
-index 4666937778bd0e25c90a47057080a5d98a697450..429bff98822e46c7a5ffcafebe2209b527d6ff47 100644
+index 4666937778bd0e25c90a47057080a5d98a697450..22a7f7908aae48cf0fc2bf0f8ed0bceb867ab115 100644
 --- a/chrome/tools/build/mac/BUILD.gn
 +++ b/chrome/tools/build/mac/BUILD.gn
 @@ -17,6 +17,7 @@ if (current_toolchain != default_toolchain) {
  
  executable("infoplist_strings_util") {
    configs += [ "//build/config/compiler:wexit_time_destructors" ]
-+  configs += [ "//brave/common:constants_configs" ]
++  configs += [ "//brave/common:constants_configs($default_toolchain)" ]
```

Comment:
Why does the toolchain matter for constants? In any case if we are touching this we should convert to a dep with a public config and put it in "//brave/tools/build/mac:infoplist_strings_util"

Reply:
(As discussed on Slack) Because we're building a tool here, which by default gets compiled with the host toolchain. In this rare case, we need Mac constants on Linux, thus need to depend on the default toolchain.